schemas/http.yml

```
@@ -15,9 +15,16 @@
       description: >
         HTTP request method.
 
-        The field value must be normalized to lowercase for querying. See
-        the documentation section "Implementing ECS".
-      example: get, post, put
+        Prior to ECS 1.6.0 the following guidance was provided:
+
+          The field value must be normalized to lowercase for
+          querying. See the documentation section "Implementing ECS".
+
+        As of ECS 1.6.0, the guidance is deprecated because the
+        original case of the method may be useful in anomaly
+        detection.  Original case will be mandated in ECS 2.0.0
```

```suggestion
        Prior to ECS 1.6.0 the following guidance was provided:

        "The field value must be normalized to lowercase, for querying."

        As of ECS 1.6.0, the guidance is deprecated because the
        original case of the method may be useful in anomaly
        detection.  Original case will be mandated in ECS 2.0.0
```

Let's fudge the quoting a bit. I think this would work best ^